backend/src/main/java/de/sambam/gamenighttracker/model/PlayerDto.java

```diff
@@ -0,0 +1,17 @@
+package de.sambam.gamenighttracker.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```

Comment:
incl. a `@builder` allows you to create incomplete objects, please keep that in mind if you rely on complete objects within your application

Reply:
yep ^^